src/js/views/admin/shared/components/published_component.js

```diff
@@ -36,7 +36,7 @@ export default Droplist.extend({
     this.setState({ selected: this.collection.get(status) });
   },
   // Overridden for flow component
-  isPublishDisabled: _.constant(false),
+  isPublishDisabled: constant(false),
```

Comment:
does our linter complain about `isPublishedDisabled: () => false,`.   Might be better in this case.